File: fib.py
# Fibonacci in python.
# @nirum Jan 6 2019


def fib(n):
    if n == 0:
        return 1
    elif n == 1:
        return 1
    else:
        return fib(n - 1) + fib(n - 2)


if __name__ == '__main__':
    for j in range(35):
        print(f'{j}\t{fib(j)}')

File: fib.lua
-- Fibonacci in lua.
-- @nirum Jan 6 2019


-- 0 1 2 3 4 5 6  ...
-- 1 1 2 3 5 8 11 ...
function fib(n)
  if n == 0 then
    return 1
  elseif n == 1 then
    return 1
  else
    return fib(n - 1) + fib(n - 2)
  end
end


for j = 0, 35 do
  print(j, fib(j))
end
